lib/features/vpn/vpn_switch.dart

```diff
@@ -110,4 +138,16 @@ class _VPNSwitchState extends State<VPNSwitch> {
       );
     }
   }
+
+  Future<void> vpnProcessForDesktop(String vpnStatus) async {
+    final vpnNotifier = Provider.of<VPNChangeNotifier>(context, listen: false);
```

Comment:
Now we are checking for errors, if errors are then we show the message and set the status.